res/map/AWBWMapConverter.py

```diff
@@ -143,6 +147,8 @@ def countryCodeToPlayerID(x):
 		'wn': 15,
 		'aa': 16,
 		'ne': 17,
+		'sc': 18,
+		'uw': 19,
```

Comment:
All right, so now we've got Alien (but dinosaurs, and from underground) and Planet of the Apes-slash-W40K orks. Should we also add their sprites if we are supporting them in map data?
Also if AWBW only goes up to 16 players, why do they have 20 different placeable civs?